test_tracing_with_deployment.py

```diff
@@ -0,0 +1,33 @@
+from starlette.requests import Request
+
+import ray
+from ray import serve
+
+from transformers import pipeline
+
+from ray.util.tracing.tracing_helper import _inject_tracing_into_class
+
+
+
+@serve.deployment(num_replicas=1, ray_actor_options={"num_cpus": 0.2, "num_gpus": 0})
+@_inject_tracing_into_class
```

Comment:
Also, we have to do this injection on all the serve deployments or else we get the same error described in the issue, where it complains with the following error.
```
TypeError: got an unexpected keyword argument '_ray_trace_ctx'
```